taxi/views.py

```diff
@@ -44,6 +63,23 @@
     model = Car
 
 
+class CarCreateView(CreateView):
```

Comment:
Consider adding `LoginRequiredMixin` to `CarCreateView` to ensure that only authenticated users can create a new car.